README.md

```diff
@@ -21,6 +21,34 @@ Moreover, with more than 20+ custom field, you can swiftly build contact forms,
 
 So weForms is an all-in-one form builder created with the global users in mind. Its minimalistic design is not only modern but also super fast and user-friendly. Everything works on a single page. Feel no delay – because your time is precious!
 
+## Development ##
+
+### Installing Dependencies ###
+Before you can use the development version of this plugin you must install the dependencies.
+
+```
+npm install
+```
+
+### Workflow ###
+Before editing or making changes check to make sure you are not editing any concatenated files. Check the Gruntfile.js in the concat: section for details.
+If you need to test or view changes run:
+
+```
+grunt concat:(specified source)
+```
+
+### Minify JS ###
+To minify Javascript for development run the following command.
```

Comment:
Why would I want to do this during development?

Reply:
How about this?

If you want to test changes with SCRIPT_DEBUG set to false. You will need to make sure the scripts are minified.
To minify Javascript for testing run the following command.

Reply:
@avonville sounds good.